appveyor.yml

```diff
@@ -17,6 +17,7 @@ install:
     Move-Item win_bison.exe bin\bison.exe -force
     Move-Item win_flex.exe bin\flex.exe -force
     Move-Item FlexLexer.h include\FlexLexer.h -force
+    Remove-Item bin\data -Force -Recurse -ErrorAction SilentlyContinue
     Move-Item data bin\data -force
```

Comment:
Is the `-force` then no longer necessary here?

Reply:
I'm not entirely sure actually. The docs seem to indicate that `-Force` suppresses user confirmation, but that's also what `-Confirm` seems to be for.

In [my testing](https://tio.run/##hZHBboMwDIbveQoLrqN7gQqpCWq3w3ZoK@04IfBGpBRXiTuKtHdnASbUUWC3OIk////vM1VoXYHGNK9YRc@MJ0i0fd9Adz7WZ2xrzJhsLYIjXhk2AXzDATlSVDKW3Dc8brXBFV9ZiD8kuUCSUyQ5R1ILJDVFUnOkZIGUTJGSgdS8Wc0YPZFjCNZrOHBqvRHKa4jjQOzaxkKbvJu16uuZmEakF/pC8LnHIDtUW98upMtGhP6LdvBJJUJJ1d3AcJQhONbGQJE6IJND1mu50yX/0aVaXUmnK4QPbR0/QI4G2Qco9ngatEZbshlCtMfsYh326Y28qN/b0GO9l9Q4WjIkL15j@za4ULM@bhbV/AA) I've not actually seen much of a difference between `Move-Item` and `Move-Item -Force`, although that might just be a property of the environment there.

Reply:
Thanks a lot for the info. So I'd remove the `-force` and wait for it to break, but I'll let the code owners make the call here.